client/client.go

```diff
@@ -96,7 +96,9 @@ type Client interface {
 	ScatterRegion(ctx context.Context, regionID uint64) error
 	// ScatterRegionWithOption scatters the specified region with the given options, should use it
 	// for a batch of regions.
-	ScatterRegionWithOption(ctx context.Context, regionID uint64, opts ...ScatterRegionOption) error
+	ScatterRegionWithOption(ctx context.Context, regionID uint64, opts ...RegionOption) error
```

Comment:
Is there possible to add this option to ScatterRegion instead of creating a new method?

Reply:
The option for Scatter and Split is much similar. If there is no repo using this api, I tend to rename it as `RegionOption`.

@overvenus Is `ScatterRegionWithOption` being used in tools?

Reply:
updated.